app/services/twitter_client/client.rb

```diff
@@ -0,0 +1,72 @@
+module TwitterClient
+  # Twitter client (users twitter gem as a backend)
+  class Client
```

Comment:
I know the various HTTP clients have code in common by now but it's still a bit early to group the variations in a common base class. There's still a bit of refactoring to be done

Reply:
100% agree, let's finish this refactor first then see what we can consolidate. That method worked great for Search.